server/CMakeLists.txt

```diff
@@ -153,6 +157,7 @@ add_llvm_executable(utbot main.cpp)
 target_link_libraries(utbot PUBLIC
         UnitTestBotLib
         loguru
+        pugixml #########
```

Comment:
No need to link with `pugixml` again since it is already linked to `UnitTestBotLib`. The same practice should be applicable to `loguru` library